react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -1,20 +1,80 @@
 import React from 'react';
 
+import { calculateWinner } from './components/utils';
 import styles from './styles.module.scss';
 import Board from './components/Board';
 
-function Game () {
-  return (
-    <div className={styles.game}>
-      <div className={styles.gameBoard}>
-        <Board />
-      </div>
-      <div className={styles.gameInfo}>
-        <div>{/* status */}</div>
-        <ol>{/* TODO */}</ol>
+class Game extends React.Component {
+  state = {
+    history: [{
+      squares: Array(9).fill(null)
+    }],
+    stepNumber: 0,
+    xIsNext: true,
+    winner: null
+  };
+
+  handleClick = (i) => {
+    const { history } = this.state;
```

Comment:
In the Line 18 you do [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) of the state `const { history } = this.state;` you should add also here all the objects with you use of the state `winner xIsNext`